presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/EnvironmentFactory.java

```diff
@@ -35,6 +36,7 @@ public EnvironmentFactory(Map<String, EnvironmentProvider> environmentProviders)
 
     public Environment.Builder get(String environmentName)
     {
+        environmentName = name(environmentName);
```

Comment:
I don't understand this. How is that effected that both hyphenated and CamelCase are accepted?

Reply:
I guess this comes from the fact that `CaseFormat.UPPER_CAMEL.to(CaseFormat.LOWER_HYPHEN, name);` is identity on name which is already LOWER_HYPHEN.

Reply:
Naaa - I do not get it too :/

Reply:
Can we make the code less "clever" or more documented?

Reply:
I will do unit tests for that.

`name()` makes formatting uniformed
 - `ala` becomes `ala`
 - `Ala` becomes `ala`
 - `DuzAla` becomes `duza-ala`
 - `duza-ala` becomes `duza-ala`
 - `Duz-Ala` becomes `duza-ala`

Reply:
> `name()` makes formatting uniformed

-> `canonicalName`  + javadoc?

Reply:
> Duz-Ala becomes duza-ala

My bad `Duz-Ala` becomes `duz--ala`

Reply:
> canonicalName

I don't think it needs a javadoc